docs/fields.asciidoc

```diff
@@ -16,6 +16,8 @@ grouped in the following categories:
 * <<exported-fields-apm-error>>
 * <<exported-fields-apm-sourcemap>>
 * <<exported-fields-apm-span>>
+* <<exported-fields-apm-span>>
+* <<exported-fields-apm-transaction>>
 * <<exported-fields-apm-transaction>>
```

Comment:
Are spans and transactions listed twice now?

Reply:
thanks :man_facepalming: added to common fields instead